app/app/views/employees/show.html.erb

```diff
@@ -1,13 +1,14 @@
 <h2> Employee Details </h2> <br>
 <%= link_to 'Back', store_path(params[:store_id]) %>  
 <%= link_to 'Edit', edit_store_employee_path(params[:store_id], params[:id]) %>
+
 <section> 
 <div> Name: <%= @employee.name %> <div/>
 <div> Email: <%= @employee.email %> <div/>
 <div> Private number: <%= @employee.private_number %> <div/>
 <div> Position: <%= @employee.position %> <div/>
 <div> Id number: <%= @employee.id_employee %> <div/>
+<div> Status: <%= @employee.status %></div>
 <section/>
```

Comment:
same this here, [like this](https://github.com/BC-MAY-21-ROR/kata-06-el-reloj-checador-k6-ca-reilero-02/pull/32/files#:~:text=All%20this%20div%20label%20are%20all%20wrong%20closing%2C%20please%20fix%20it%20like%20this%20way%20%2D%3E%20%3C/div%3E)